### PR TITLE
Add latest version tag for service on service order page.

### PR DESCRIPTION
**Pull request by WangLiNaruto**
fix  eclipse-xpanse/xpanse#221

image

### CODE REVIEW

File: src/components/content/order/Services.tsx [hunk 1]
```diff
@@ -31,18 +35,27 @@ function Services(): JSX.Element {
...
         serviceVendorApi.listRegisteredServicesTree(categoryName).then((rsp) => {
             if (rsp.length > 0) {
-                let serviceList: { name: string; content: string; icon: string }[] = [];
+                let serviceList: { name: string; content: string; icon: string; defaultVersion: string }[] = [];
                 rsp.forEach((item) => {
                     let serviceItem = {
                         name: item.name || '',
                         content: item.versions[0].cloudProvider[0].details[0].description,
                         icon: item.versions[0].cloudProvider[0].details[0].icon,
+                        defaultVersion: item.versions.sort((v1, v2) => {
```
Comment (iskey) [inline src/components/content/order/Services.tsx:44]: can we refine the version sort? it seems v1.9 is newer than v.1.10 by this implementation.

File: src/components/content/order/Services.tsx [hunk 2]
```diff
@@ -9,18 +9,22 @@ import { useEffect, useState } from 'react';
...
 function Services(): JSX.Element {
-    const [services, setServices] = useState<{ name: string; content: string; icon: string }[]>([]);
+    const [services, setServices] = useState<{ name: string; content: string; icon: string; defaultVersion: string }[]>(
+        []
+    );
     const navigate = useNavigate();
     const location = useLocation();
 
-    const onClicked = function (cfg: string) {
+    const onClicked = function (cfg: string, defaultVersion: string) {
```
Comment (iskey) [inline src/components/content/order/Services.tsx:22]: also, is it better to rename the defaultVersion to latestVersion?

**Review by iskey — CHANGES_REQUESTED** [review]
comments added.

**Comment by iskey** [comment]
conflicts must be resolved.

File: src/styles/service_order.css [hunk 3]
```diff
@@ -111,14 +110,13 @@
...
     height: 100px;
-    margin: 10px;
     border: 1px solid #1677ff;
 }
 
 .cloud-provider-select {
     display: flex;
     flex-flow: nowrap row;
-    padding: 0;
+    padding: 0px;
```
Comment (iskey) [inline src/styles/service_order.css:119]: `px` is not needed here.

File: src/styles/service_order.css [hunk 4]
```diff
@@ -129,21 +127,22 @@
     cursor: pointer;
     display: flex;
     flex-flow: nowrap row;
-    padding: 0;
+    padding: 0px;
```
Comment (iskey) [inline src/styles/service_order.css:130]: ditto

File: src/styles/service_order.css [hunk 5]
```diff
@@ -129,21 +127,22 @@
...
     border-radius: 4%;
+    /*background:#795548;*/
 }
 
 .cloud-provider-select:hover {
     cursor: pointer;
     display: flex;
     flex-flow: nowrap row;
-    padding: 0;
+    padding: 0px;
```
Comment (iskey) [inline src/styles/service_order.css:145]: doitto.

File: src/styles/service_order.css [hunk 6]
```diff
@@ -224,8 +223,6 @@
     margin: 0 auto;
 }
 
-.submit-alert-tip {
-    margin-bottom: 20px;
-    margin-left: 30px;
-    margin-right: 30px;
+.services-content-body-space {
```
Comment (iskey) [inline src/styles/service_order.css:226]: I think the `submit-alert-tip` should be kept.

**Comment by WangLiNaruto** [comment]
image

**Review by iskey — APPROVED** [review]
looks good.

**Pull request merged**